docs/guides/tools-reference/tools/rclone-object-storage-file-sync/index.md

```diff
@@ -59,17 +60,17 @@ Before you configure Rclone, [create a new Linode bucket](/docs/products/storage
 
 1. Use a human-readable name as the configuration name.
 
-1. Then, select your cloud storage service. In this case, select AWS S3 Compliant Storage Provider (option 4), followed by selecting any other S3 compatible provider (option 14).
+1. Then, select your cloud storage service. In this case, select AWS S3 Compliant Storage Provider, followed by selecting any other S3 compatible provider.
 
-1. Get AWS credentials. For this step, type `2` and then type your **Access Key** string (for `access_key_id`) and then paste your Secret Key for `secret_access_key`.
+1. Get AWS credentials. For this step, type the number that corresponds to the option **Get AWS credentials from the environment (env vars or IAM)** and then type your **Access Key** string (for `access_key_id`) and then paste your Secret Key for `secret_access_key`.
 
 1. Leave the `region` section blank.
 
 1. For the endpoint, type the address for your Linode bucket, such as `rclone.us-east-1.linodeobjects.com`.
 
 1. Leave `location_constraint` blank.
 
-1. Select the `acl` you'd like to use. If this is a personal bucket, type `1` for private. If this is a team bucket, you might select `public-read` (option 2) or `authentication-read` (option 4).
+1. Select the `acl` you'd like to use. If this is a personal bucket, type the number that corresponds to the option **No one else has access rights (default) \ (private)**. If this is a team bucket, you might select `public-read` option or `authentication-read` option.
```

Comment:
I fixed the bold formatting and also removed the extra line and period. Is this now the correct output?

> **No one else has access rights (default) \ (private)**

If so, we can merge it in.

Reply:
Hi @wildmanonline Here's the terminal output.
<img width="675" alt="Screenshot 2023-07-27 at 4 55 39 PM" src="https://github.com/linode/docs/assets/37059749/330fd9c8-923f-42a2-aee2-d3bdc9820219">

Reply:
Ah, I see. Thanks! I've updated that step and, since users might want any of those ACLs, I linked out to our ACL guide.